### ISSUE TITLE
Help widget

### ISSUE THREAD

**Issue by tlongers** [security-force-monitor/sfm-cms#278]
Something like this (sorry for poor diagram):
![ux - help widgets on content areas](https://user-images.githubusercontent.com/2115933/31025691-68bceb48-a53b-11e7-8cf6-d29609108664.png)
- place a "?" in the title of every content area
- on click, head to relevant section on help.~

**Comment by jeancochrane** [comment]
@tlongers The widget is ready to be implemented whenever the research handbook is in a good enough state (to me, that means that relevant section permalinks won't change). Give me the word and I'll jump on it!

**Comment by tlongers** [comment]
Got it. Bear with!

**Comment by tlongers** [comment]
V nearly there with this :)

**Comment by jeancochrane** [comment]
Excited to see it!

**Comment by tlongers** [comment]
Herewith, after some remarkably tedious issues with GitBook (it seems to hate certain `xattr` tags in OSX files and won't build branches correctly until merged - I don't get it!) some documentation we can wire in to the record pages:

* Units: https://help.securityforcemonitor.org/en/whowasincommand/unitrec.html
* Persons: https://help.securityforcemonitor.org/en/whowasincommand/personsrec.html

I've structured the documentation by the content section of the record, and should be fairly stable now.  Each bit of documentation has an image of the part of the page it documents, a short explanation of what's going on, and an overview of the data fields used (along with link to the field-specific documentation). All sections headings have a specified anchor.

So, for the Chain of Command section in Person records the help link should go direct to the relevant anchor:

https://help.securityforcemonitor.org/en/whowasincommand/personsrec.html#person_record_anatomy_chain_of_command

The documentation is also internationalized: in `en` and `es` as with the platform. The `es` version is not yet translated but will be - if the user is looking at an `es` record it would be awesome if the help widget linked out to the `es` version of the help.

I've also written a more detailed FAQ here, which we need to figure out how to incorporate:

https://help.securityforcemonitor.org/en/whowasincommand/generalfunctions.html

Still to do: Events/incidents

**Comment by jeancochrane** [comment]
Amazing! I'll go ahead and put in help widgets for units and personnel.

**Comment by tlongers** [comment]
Documentation containing an overview  of Incidents is [now done too](https://help.securityforcemonitor.org/en/whowasincommand/eventrec.html).

**Comment by jeancochrane** [comment]
Alright, help widgets are in as soon as https://github.com/security-force-monitor/sfm-cms/commit/2c8650e05d2bf2d37d7ec89d8399bb104646f2b7 deploys to staging.

I wired up everything for personnel/units/incidents as expected. I also tried to put in as many links to FAQ items as I could, meaning that a help widget shows up now for:

- No results found
- Suggested searches
- Source/confidence popover (next to the "dismiss" button)
- Real start/end date popover (next to a new "dismiss" button)

Let me know if I missed anything obvious; otherwise, I'm ready to close this when you are!
